server/controllers/configurationsController/index.js

```diff
@@ -57,50 +55,48 @@ const ConfigurationsController = {
     }
   },
   index: async (req, res) => {
-    try {
-      const dataSet = new Set()
-      const { appDb } = req.app.locals
-      const { uid } = req.params
-      const streamConfigurations = await ConfigModel.index(appDb, uid)
+    const cachedUsers = new Map()
+    const dataSet = new Set()
+    const { appDb } = req.app.locals
+    const { uid } = req.params
+    const activeOnly = Object.hasOwn(req.query, 'activeOnly')
+    const streamConfigurations = await ConfigModel.all(appDb, uid, activeOnly)
```

Comment:
I think we should use a different function for the active configurations so we're not adding more and more logic to the `all` function over time.

```suggestion
    const streamConfigurations = activeOnly ? await ConfigModel.active(appDb, uid) : await ConfigModel.all(appDb, uid)
```

Reply:
I want to make sure it's clear on my end. Would using active as a method in the model deviate from the convention that you are following..? I ask because in the past when I've added a method that isn't all, upsert, create etc... it was considered incorrect.

Reply:
nope - it's not deviating from the convention to add other methods, it deviates from the convention to name the `all` method `index` (which is the controller action and route name).

Reply:
[2afe7af](https://github.com/AMP-SCZ/dpdash/pull/739/commits/2afe7af8b409f3ec568dc3f205c9af37c68d334e)